### PR TITLE
Stewart: refactor patterns

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Stewart: refactor patterns following the latest Gutenberg changes

#### Related issue(s):
https://github.com/Automattic/themes/issues/5803

### CODE REVIEW

File: stewart/patterns/header-image.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Title: Header image block pattern
```
Comment (pbking) [inline stewart/patterns/header-image.php]: Title should just be 'Header image'

Comment (pbking) [inline stewart/patterns/header-image.php]: Title should just be 'Header image'

File: stewart/patterns/header-traditional.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Title: Traditional header block pattern
```
Comment (pbking) [inline stewart/patterns/header-traditional.php]: just 'Traditional header`

Comment (pbking) [inline stewart/patterns/header-traditional.php]: just 'Traditional header`

File: stewart/patterns/page-about.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Title: About page block pattern
```
Comment (pbking) [inline stewart/patterns/page-about.php]: Just 'About page'

Comment (pbking) [inline stewart/patterns/page-about.php]: Just 'About page'

File: stewart/patterns/page-contact.php [hunk 4]
```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Title: Contact page block pattern
```
Comment (pbking) [inline stewart/patterns/page-contact.php]: just 'Contact page'

Comment (pbking) [inline stewart/patterns/page-contact.php]: just 'Contact page'

File: stewart/patterns/posts-list.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Simple list of posts
```
Comment (pbking) [inline stewart/patterns/posts-list.php]: Title should be 'Posts list'

Comment (pbking) [inline stewart/patterns/posts-list.php]: Title should be 'Posts list'

File: stewart/patterns/posts-list.php [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * Slug: stewart/posts-list
+ * Categories: featured, query
```
Comment (pbking) [inline stewart/patterns/posts-list.php]: wasn't previously in 'featured' category

Comment (pbking) [inline stewart/patterns/posts-list.php]: wasn't previously in 'featured' category

File: stewart/patterns/sidebar-background.php [hunk 7]
```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Title: Sidebar with background color block pattern
```
Comment (pbking) [inline stewart/patterns/sidebar-background.php]: Title just: 'Sidebar with background color'

Comment (pbking) [inline stewart/patterns/sidebar-background.php]: Title just: 'Sidebar with background color'

File: stewart/patterns/sidebar-default.php [hunk 8]
```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Title: Default sidebar block pattern
```
Comment (pbking) [inline stewart/patterns/sidebar-default.php]: title just `Default sidebar`

Comment (pbking) [inline stewart/patterns/sidebar-default.php]: title just `Default sidebar`

File: stewart/patterns/sidebar-categories-tags.php [hunk 9]
```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Title: Sidebar with categories and tags block pattern
```
Comment (pbking) [inline stewart/patterns/sidebar-categories-tags.php]: Title just 'Sidebar with categories and tags'

Comment (pbking) [inline stewart/patterns/sidebar-categories-tags.php]: Title just 'Sidebar with categories and tags'

File: stewart/patterns/sidebar-blogging.php [hunk 10]
```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Title: Blogging sidebar block pattern
```
Comment (pbking) [inline stewart/patterns/sidebar-blogging.php]: title just 'Blogging sidebar'

Comment (pbking) [inline stewart/patterns/sidebar-blogging.php]: title just 'Blogging sidebar'

File: stewart/patterns/header-image.php [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+ * Slug: stewart/header-image
+ * Categories: header
```
Comment (pbking) [inline stewart/patterns/header-image.php:5]: should have header blocktype added to metadata

Comment (pbking) [inline stewart/patterns/header-image.php:5]: should have header blocktype added to metadata

File: stewart/patterns/header-traditional-background.php [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Title: Traditional header with color background
+ * Slug: stewart/header-traditional-background
+ * Categories: header
```
Comment (pbking) [inline stewart/patterns/header-traditional-background.php:5]: Should have header blocktype added

Comment (pbking) [inline stewart/patterns/header-traditional-background.php:5]: Should have header blocktype added

File: stewart/patterns/header-traditional.php [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+ * Slug: stewart/header-traditional
+ * Categories: header
```
Comment (pbking) [inline stewart/patterns/header-traditional.php:5]: Should have header blocktype added

Comment (pbking) [inline stewart/patterns/header-traditional.php:5]: Should have header blocktype added

File: stewart/patterns/footer-right.php [hunk 14]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Title: Right-aligned footer block pattern
```
Comment (pbking) [inline stewart/patterns/footer-right.php]: Title just 'Right-aligned footer`

Comment (pbking) [inline stewart/patterns/footer-right.php]: Title just 'Right-aligned footer`

File: stewart/patterns/footer-nav-right.php [hunk 15]
```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Footer with right navigation block pattern
```
Comment (pbking) [inline stewart/patterns/footer-nav-right.php]: Title just 'Footer with right navigation'

Comment (pbking) [inline stewart/patterns/footer-nav-right.php]: Title just 'Footer with right navigation'

File: stewart/patterns/footer-nav-left.php [hunk 16]
```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Footer with left navigation block pattern
```
Comment (pbking) [inline stewart/patterns/footer-nav-left.php]: Title shouldn't have 'block pattern' in it.

Comment (pbking) [inline stewart/patterns/footer-nav-left.php]: Title shouldn't have 'block pattern' in it.

File: stewart/patterns/footer-left.php [hunk 17]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Title: Default footer block pattern
```
Comment (pbking) [inline stewart/patterns/footer-left.php]: ~~block pattern~~

Comment (pbking) [inline stewart/patterns/footer-left.php]: ~~block pattern~~

File: stewart/patterns/footer-default.php [hunk 18]
```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Default footer block pattern
```
Comment (pbking) [inline stewart/patterns/footer-default.php]: Title should be 'Default footer ~~block pattern~~'

Comment (pbking) [inline stewart/patterns/footer-default.php]: Title should be 'Default footer ~~block pattern~~'

**Comment by pbking** [comment]
Looks like most of the titles (but not all) had 'block pattern' appended to them (and shouldn't).  Also the header patterns should have the block type added to the metadata.

The actually implementation looks solid though, I didn't notice any discrepancies there. 

Just need to fix up some of the metadata.

**Comment by matiasbenedetto** [comment]
The small fixes pointed out by @pbking were solved.
Thanks for the review!

**Review by mikachan — APPROVED** [review]
Looks like all @pbking's comments have been addressed. LGTM! 🚢

**Pull request merged**